.flake8

```diff
@@ -1,5 +1,5 @@
 [flake8]
-ignore = E111,E114,E501,E261,E266,E121,E402,E241,W504
+ignore = E111,E114,E501,E261,E266,E121,E402,E241,W504,E741
```

Comment:
Do we have a lot of variables named `I`, `l`, or `O`? If there are not too many, it might be good to rename them.

Reply:
Ah yes, we have a lot of `[l for l in lines ...]` comprehensions that triggers this.. and I think that is a little overzelous.

Reply:
I guess we could change to `[ln for ln in lines ..]`?

Reply:
oh yeah, I do like having single-letter variables in comprehensions. It's too bad we can't apply this lint everywhere except in comprehensions. I'm fine either way.